internal/pkg/template/templates/workloads/partials/cf/addons.yml

```diff
@@ -11,5 +11,8 @@ AddonsStack:
       App: !Ref AppName
       Env: !Ref EnvName
       Name: !Ref WorkloadName
+      {{- if .AddonsExtraParams }}
+{{ .AddonsExtraParams | indent 6 }}
```

Comment:
do we not set the indent here 🤔  https://github.com/aws/copilot-cli/pull/2978/files#diff-cc1f6de0ed3926d7713d70861fd635ec7ff5fccaa08d0d7d308411990761bf7cR144

Reply:
They're different indentations 😄 

the one you linked allow us to marshal this way:
```yaml
Service:
   !Ref Service # has 2 spaces instead of yaml.Marshal default of 4
```

whereas this indent takes the whole block above and adds 6 spaces before it:
```yaml
      Service: # same as above but completely shifted by 6 spaces
        !Ref Service
```

Reply:
right right